cmd/server/assets/header.html

```diff
@@ -55,39 +55,41 @@
     </div>
   {{end}}
 
-  <nav class="nav nav-tabs navbar-expand-md navbar-light bg-light">
```

Comment:
This is just indented (?w=1). It can really only fail in tests, but it's good to guard against it anyway in the template.